src/aleph/web/__init__.py

```diff
@@ -8,53 +8,67 @@
 import pkg_resources
 import socketio
 from aiohttp import web
+
 from aleph.web.controllers.routes import register_routes
 
-app = web.Application(client_max_size=1024 ** 2 * 64)
-sio = socketio.AsyncServer(async_mode="aiohttp", cors_allowed_origins="*")
-auth = None
-
-# Configure default CORS settings.
-cors = aiohttp_cors.setup(
-    app,
-    defaults={
-        "*": aiohttp_cors.ResourceOptions(
-            allow_methods=["GET", "POST"],
-            allow_credentials=True,
-            expose_headers="*",
-            allow_headers="*",
-        )
-    },
-)
-sio.attach(app)
-
-tpl_path = pkg_resources.resource_filename("aleph.web", "templates")
-JINJA_LOADER = jinja2.ChoiceLoader(
-    [
-        jinja2.FileSystemLoader(tpl_path),
-    ]
-)
-aiohttp_jinja2.setup(app, loader=JINJA_LOADER)
-env = aiohttp_jinja2.get_env(app)
-env.globals.update(
-    {
-        "app": app,
-        "date": date,
-        "datetime": datetime,
-        "time": time,
-        "timedelta": timedelta,
-        "int": int,
-        "float": float,
-        "len": len,
-        "pprint": pprint,
-    }
-)
-
-register_routes(app)
-
-
-def init_cors():
+
+def init_cors(app: web.Application):
+    # Configure default CORS settings.
+    cors = aiohttp_cors.setup(
+        app,
+        defaults={
+            "*": aiohttp_cors.ResourceOptions(
+                allow_methods=["GET", "POST"],
+                allow_credentials=True,
+                expose_headers="*",
+                allow_headers="*",
+            )
+        },
+    )
+
     # Configure CORS on all routes.
     for route in list(app.router.routes()):
         if "/socket.io/" not in repr(route.resource):
             cors.add(route)
+
+
+def init_sio(app: web.Application) -> socketio.AsyncServer:
+    sio = socketio.AsyncServer(async_mode="aiohttp", cors_allowed_origins="*")
+    sio.attach(app)
+    return sio
+
+
+def create_app() -> web.Application:
+    app = web.Application(client_max_size=1024 ** 2 * 64)
+
+    tpl_path = pkg_resources.resource_filename("aleph.web", "templates")
+    jinja_loader = jinja2.ChoiceLoader(
+        [
+            jinja2.FileSystemLoader(tpl_path),
+        ]
+    )
+    aiohttp_jinja2.setup(app, loader=jinja_loader)
+    env = aiohttp_jinja2.get_env(app)
+    env.globals.update(
+        {
+            "app": app,
+            "date": date,
+            "datetime": datetime,
+            "time": time,
+            "timedelta": timedelta,
+            "int": int,
+            "float": float,
+            "len": len,
+            "pprint": pprint,
+        }
+    )
+
+    register_routes(app)
+
```

Comment:
Too many empty lines to my taste, non-blocking 😉.

Reply:
How am I gonna increase my stats on Github otherwise?